docs/edgeql/sdl/index.rst

```diff
@@ -13,13 +13,19 @@ language and the imperative low-level :ref:`DDL <ref_eql_ddl>`.
 
 SDL is a declarative language optimized for human readability and
 expressing the state of the EdgeDB schema without getting into the
-details of how to arrive at that state.  Each *.esdl* file represents
-a complete schema state for a particular
-:ref:`module <ref_datamodel_modules>`.
+details of how to arrive at that state.  Each *SDL* block represents
+the complete schema state for a given :ref:`database
+<ref_datamodel_databases>`.
 
 Syntactically, an SDL declaration mirrors the ``CREATE`` DDL for the
 corresponding entity, but with all of the ``CREATE`` and ``SET``
-keywords omitted.
+keywords omitted.  These individual declarations must either use
+:ref:`fully-qualified names <ref_eql_fundamentals_name_resolution>` or
+can be enclosed into a *module block* where only short names are
```

Comment:
`*module block*` should ref `ref_eql_sdl_modules`

Reply:
OK